spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -3,95 +3,69 @@
 include Features::ActiveJobHelper
 
 feature 'Two Factor Authentication' do
-  describe 'When the user has not setup 2FA' do
-    scenario 'user is prompted to setup two factor authentication at first sign in' do
-      sign_in_before_2fa
+  describe 'When the user has not set up 2FA' do
+    scenario 'user is prompted to set up two factor authentication at account creation' do
+      user = sign_in_before_2fa
+
+      # user attempts to circumnavigate 2FA setup
+      visit profile_path
 
       expect(current_path).to eq phone_setup_path
       expect(page).
         to have_content t('devise.two_factor_authentication.two_factor_setup')
-    end
 
-    scenario 'user does not fill out a phone number when signing up' do
-      sign_up_and_set_password
+      # user clicks "send passcode" without entering a phone number
       click_button t('forms.buttons.send_passcode')
```

Comment:
What I did here is consolidate separate tests into one to speed up this feature spec. It feels weird to me to define methods that are only used by one test. They wouldn't be reusable since they depend on a specific context. If I have a method called `click_passcode_without_entering_phone_number`, I would only be able to use it in a specific context, but maybe that's OK? I suppose if they're only defined here, they would only be reusable in this file.